src/app/f2f/controllers/landingPage.js

```diff
@@ -12,6 +12,10 @@ class LandingPageController extends BaseController {
         // Show thin file user screen
         req.sessionModel.set("isThinFileUser", true);
       }
+      if (configData && configData.pcl_enabled) {
```

Comment:
```suggestion
      if (configData) {
      // Check if configData.pcl_enabled is defined
      req.sessionModel.set("pclEnabled", !!configData.pcl_enabled);
     }
```

Reply:
the value returned is a string ("true"/"false") and not as a boolean. so updated the code to handle this. Have tested this. it will be set to false if the attribute is missing/undefined and set to the true if the string value matches true else set to false.